apps/client/src/common/components/input/colour-input/Swatch.tsx

```diff
@@ -25,5 +25,11 @@ export default function Swatch(props: SwatchProps) {
       </div>
     );
   }
+
+  if (color === 'unkown') {
+    <div className={`${classes} ${style.center}`} onClick={handleClick}>
+      ?
+    </div>;
+  }
```

Comment:
I'm unsure about this
I think that an unknown color should result i nothing being highlighted in the swatch
or dose it serve a function?